packages/slice-machine/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -206,3 +216,27 @@ export const WithBoxContent = {
   },
   parameters: { controls: { hideNoControlsWarning: true, include: [] } },
 };
+
+type OrganizationSelectProps = { disabled?: boolean };
+
+function OrganizationSelect(props: OrganizationSelectProps): ReactNode {
+  return (
+    <Select
+      {...props}
+      color="grey"
+      constrainContentWidth
+      flexContent
+      placeholder="Organization"
+      renderStartIcon={() => <GitHubIcon color={tokens.color.greyLight11} />}
+      size="large"
+      sx={{ width: "calc(50% - 8px)" }}
+    >
+      <SelectItem size="large" value="organization-1">
+        Organization
```

Comment:
We may want to add icons to the select items to represent the Git provider. In this case, we would probably want a GitHub logo to appear next to each item. In the future, we would show different provider icons, like the GitLab and Bitbucket logos.

Reply:
We should probably support multiple connected accounts at once, like the Vercel "New Project" screen works:

<img width="400" alt="image" src="https://github.com/prismicio/slice-machine/assets/8601064/c9ac6fbd-45d2-4000-89ea-7d81f6c46ca8">

They force switching Git providers, but I think we can support multiple providers at once. Thus, we would need to a way to differentiate providers.

I don't think owner type is as important as provider type since a user may add two accounts with the same name from different providers. The different between a user and team owner has less of an impact, but we could still display a "team" label next to team accounts.